content/docs/integrations/gcalendar/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Google Calendar Plugin 
+publishedDate: '2022-03-29T14:00:00.0Z'
+description: How to use the Google calendar plugin
+---
+
+## Introduction
+
+The Google Calendar plugin provides a home page card you can use to display your agenda in Backstage.
+
+![gcp-api-search](./gcalendar-card.png)
+
+## Installation
+
+1. Set up google auth in Roadie following [this guide](/docs/integrations/google-oauth-client)
+
+2. Ensure the Google calendar API is enabled in GCP.
```

Comment:
I think it's worth moving these into headings rather than having a nested numbered list.

I can see it working like:

h2 Set up Google OAuth

Set up google auth in Roadie following [this guide](/docs/integrations/google-oauth-client)

h2. Add the Google Calendar card

Steps 1, 2 and 3.

Reply:
Yep, I'll do that.